### PR TITLE
Files in values and values-dev

### PR DESCRIPTION
**Pull request by joelstobart-moj**

### CODE REVIEW

File: src/main/resources/application-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Most values are inherited from - application.yml - except the following:
+
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          public-key-location: classpath:local-public-key.pub
+
```
Comment (timharrison-moj) [inline src/main/resources/application-dev.yaml:15]: Probably need a copy of this file `local-public-key.pub` from CVL? It goes into `src/main/resources`

**Review by timharrison-moj — COMMENTED** [review]
LGTM - one comment only. Can go in now or later, just wondering if it would start up with that missing though?

File: src/main/resources/application-dev.yaml (hunk 1 above)
Comment (joelstobart-moj) [reply src/main/resources/application-dev.yaml:15]: I've addressed that one, also fixed an indenting issue in the application.yml

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**